calico_containers/calico_ctl/node.py

```diff
@@ -536,24 +543,31 @@ def warn_if_hostname_conflict(ip):
     """
     # If there's already a calico-node container on this host, they're probably
     # just re-running node to update one of the ip addresses, so skip..
-    if not docker_client.containers(filters={'name': 'calico-node'}):
-        # Otherwise, check if another host with the same hostname
-        # is already configured
-        try:
-            current_ipv4, _ = client.get_host_bgp_ips(hostname)
-        except KeyError:
-            # No other machine has registered configuration under this hostname.
-            # This must be a new host with a unique hostname, which is the
-            # expected behavior.
-            pass
-        else:
-            if current_ipv4 != "" and current_ipv4 != ip:
-                print_paragraph("WARNING: Hostname '%s' is already in use "
-                                "with IP address %s. Calico requires each compute host to "
-                                "have a unique hostname. If this is your first time "
-                                "running 'calicoctl node' on this host, ensure that "
-                                "another host is not already using the "
-                                "same hostname." % (hostname, ip))
+    try:
+        if docker_client.containers(filters={'name': 'calico-node'}):
+            return
+    except IOError:
```

Comment:
I think we still want to do the hostname conflict warning, even if the user doesn't have Docker running.